src/main/java/wellnus/atomichabit/command/AddCommand.java

```diff
@@ -126,7 +130,7 @@ public void execute() throws AtomicHabitException {
      * @throws BadCommandException If the arguments have any issues
      */
     @Override
-    public void validateCommand(HashMap<String, String> arguments) throws BadCommandException {
+    public void validateCommand(HashMap<String, String> arguments) throws BadCommandException, NumberFormatException {
```

Comment:
The JavaDoc does not match the function signature. What happens when the NumberFormatException gets thrown? Is it properly handled?

Reply:
Sorry forgot to remove when i was deleting my semi-implementation of not allowing integers as habit names, will update the removal.